src/load-fixtures.coffee

```diff
@@ -1,5 +1,5 @@
-`#!/usr/bin/env node
-`
```

Comment:
@aaronbuchanan : I've added these backquotes in order to keep the shebang line after coffee->js compilation

Could you remove this from your PR please ?